api/src/__init__.py

```diff
@@ -0,0 +1,121 @@
+            "Accept": "application/vnd.github+json"
+        }
+
+    def fetch_issues(self, owner: str, repo: str) -> List[GitHubIssue]:
+        url = f"https://api.github.com/repos/{owner}/{repo}/issues"
+        response = requests.get(url, headers=self.headers)
+        response.raise_for_status()
+        return [
+            GitHubIssue(
+                issue_id=item["number"],
+                title=item["title"],
+                body=item.get("body", ""),
+                url=item["html_url"],
+                state=item["state"]
+            )
+            for item in response.json()
+        ]
+
+    def close_issue(self, owner: str, repo: str, issue_id: int) -> bool:
+        url = f"https://api.github.com/repos/{owner}/{repo}/issues/{issue_id}"
+        data = {"state": "closed"}
+        response = requests.patch(url, headers=self.headers, json=data)
+        return response.status_code == 200
+
+    def comment_on_issue(self, owner: str, repo: str, issue_id: int, comment: str) -> bool:
+        url = f"https://api.github.com/repos/{owner}/{repo}/issues/{issue_id}/comments"
+        response = requests.post(url, headers=self.headers, json={"body": comment})
+        return response.status_code == 201
+
+
+# ---------- Trello API Classes ----------
+
+class TrelloCard:
+    def __init__(self, card_id, name, desc):
+        self.id = card_id
+        self.name = name
+        self.desc = desc
+
+class TrelloAPI:
+    def __init__(self, key: str, token: str):
+        self.key = key
+        self.token = token
+
+    def create_card(self, list_id: str, card: TrelloCard) -> bool:
+        url = "https://api.trello.com/1/cards"
+        params = {
+            "key": self.key,
+            "token": self.token,
+            "idList": list_id,
+            "name": card.name,
+            "desc": card.desc
+        }
+        response = requests.post(url, params=params)
+        return response.status_code == 200
+
+    def move_card_to_list(self, card_id: str, list_id: str) -> bool:
+        url = f"https://api.trello.com/1/cards/{card_id}"
+        params = {
+            "key": self.key,
+            "token": self.token,
+            "idList": list_id
+        }
+        response = requests.put(url, params=params)
+        return response.status_code == 200
+
+    def get_cards_from_list(self, list_id: str) -> List[TrelloCard]:
+        url = f"https://api.trello.com/1/lists/{list_id}/cards"
+        params = {
+            "key": self.key,
+            "token": self.token
+        }
+        response = requests.get(url, params=params)
+        response.raise_for_status()
+        return [
+            TrelloCard(
+                card_id=item["id"],
+                name=item["name"],
+                desc=item["desc"]
+            )
+            for item in response.json()
+        ]
+
+
+# ---------- Sync Manager ----------
+
+class IssueSyncManager:
+    def __init__(self, github_api: GitHubAPI, trello_api: TrelloAPI):
```

Comment:
There seems to be a confusion on the goal of the project. 
Why is sync relevant? We want to create an interface that defines a generic issue tracker client. We don't care about how the data gets to the server.